west.yml

```diff
@@ -22,9 +22,16 @@ manifest:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
 
+  group-filter: [-babblesim]
+
   #
   # Please add items below based on alphabetical order
   projects:
+    - name: bsim
+      repo-path: babblesim-manifest
```

Comment:
needs further discussion, this manifest repo includes repos not hosted by the zephyr project, something that is not covered by the current guidelines and rules for includng repos in the manifest.

Reply:
Can we set up a Zephyrproject mirror for the repo? Who do we need to talk to? @carlescufi, do you know?